src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -104,17 +104,18 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
   }
 
   saveEntry(event: SaveEntryEvent): void {
-    if (this.activeTimeEntry) {
+
+    if (this.activeTimeEntry && this.entryId === this.activeTimeEntry.id) {
       const startDateAsLocalDate = new Date(event.entry.start_date);
       const endDateAsLocalDate = new Date(event.entry.end_date);
       const activeEntryAsLocalDate = new Date(this.activeTimeEntry.start_date);
-      const isEditingEntryEqualToActiveEntry = this.entryId === this.activeTimeEntry.id;
```

Comment:
I wouldn't remove this variable... I would put at the top of the method and use it instead of the second validation in the first if clause

Reply:
What happened to this comment? I see that you just get rid of the variable and you are not longer validating it. Is it because we don't need to verify if the `this.activeTimeEntry` is the same as `this.entryId`?